service/service.go

```diff
@@ -24,6 +24,7 @@ type Port struct {
 	Type        string
 	Port        int64
 	ServicePort int64
+	IP          string
```

Comment:
Because they are only used to write out text so doing that would just require parsing and generating strings over and over